### ISSUE TITLE
Rewrite proxy-to-origin coalescing case

### ISSUE THREAD

**Issue by rmarx** [kazuho/draft-kazuho-httpbis-priority#99]
Discussed in Singapore. 

Common modus operandi should be that proxies generally keep the client-specified priorities intact when forwarding. If they do, they should also set a separate signal to let the origin know it's behind a proxy that's coalescing multiple clients on 1 connection. We should also specify this signal (related to #57 but different, cc @MikeBishop). We then also need text specifying how the origin should respond to that signal (i.e., we can get rid of #77, cc @kazuho).

Should also discuss the case that if proxies want to change the priority, they should use a PRIORITY_UPDATE frame immediately after the HEADERS (solves the "how do we communicate original priorities to the origin" issue).

**Comment by c-taylor** [comment]
This issue has probably several linked parts:

> Common modus operandi should be that proxies generally keep the client-specified priorities intact when forwarding. 

I think this is now covered by #110 

> If they do, they should also set a separate signal to let the origin know it's behind a proxy that's coalescing multiple clients on 1 connection. We should also specify this signal (related to #57 but different, cc @MikeBishop). We then also need text specifying how the origin should respond to that signal (i.e., we can get rid of #77, cc @kazuho).

We discussed not specifying exactly and #110 gives examples. Authentication, session information etc.

> Should also discuss the case that if proxies want to change the priority, they should use a PRIORITY_UPDATE frame immediately after the HEADERS (solves the "how do we communicate original priorities to the origin" issue).

Leaving for @kazuho / @LPardue

**Comment by rmarx** [comment]
I don't agree we discussed not to have a separate signal. The text in the HTTP/1.1 case is good and as I understood it, but I think we need something explicit for H2 and H3 when coalescing multiple clients. cc @MikeBishop. 

If you don't specify this for H2 and H3, there's imo a low chance origin servers will actually do anything intelligently in this case.

**Comment by c-taylor** [comment]
Sorry, I used a contraction: A separate signal is required, but I think we agreed not to specify that signal and just to highlight that one was required.

**Comment by rmarx** [comment]
In that case, I would like us to revisit that agreement :) (not necessarily for the current version of course)

**Comment by kazuho** [comment]
I do not think we need to resolve this in the next draft. Therefore added a TODO to the I-D, referencing this issue from there.
